benchmarks/scripts/build_dgl_asv.sh

```diff
@@ -11,11 +11,7 @@ pip install -r /asv/torch_gpu_pip.txt
 # 'CUDA_TOOLKIT_ROOT_DIR' is always required for sparse build as torch1.13.1+cu116 is installed.
 CMAKE_VARS="-DUSE_OPENMP=ON -DBUILD_TORCH=ON -DCUDA_TOOLKIT_ROOT_DIR=/usr/local/cuda"
 if [[ $DEVICE == "gpu" ]]; then
-    CMAKE_VARS="-DUSE_CUDA=ON -DUSE_NCCL=ON $CMAKE_VARS"
-fi
-arch=`uname -m`
-if [[ $arch == *"x86"* ]]; then
```

Comment:
is AVX available on non-x86 machines?

Reply:
`USE_AVX` was removed in #5165. I guess https://github.com/dmlc/dgl/blob/866c70dace72c8d73e2011b8881fcf0a6eea4be7/src/runtime/config.cc#L20 will return false on non-AVX CPUs.

Reply:
`-DUSE_AVX=OFF` is specified when building package for osx. But does `USE_AVX` really take effect as it's already removed. @BarclayII do you have any comments on this?

Reply:
confirmed offline, `USE_AVX` does not take effect now. so it's ok to remove it at all.